expression/builtin_miscellaneous.go

```diff
@@ -221,7 +224,39 @@ type builtinInetNtoaSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_inet-ntoa
 func (b *builtinInetNtoaSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("INET_NTOA")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+
+	arg := args[0]
+	if arg.IsNull() {
+		d.SetNull()
+		return d, nil
+	}
+
+	ipArg, err := arg.ToInt64(b.ctx.GetSessionVars().StmtCtx)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	d.SetNull()
+	uIPArg := uint64(ipArg)
+	if uIPArg > 0xffffffff {
```

Comment:
Can we first check whether `uint64(ipArg) > 0xffffffff`, then convert the type of `ipArg` to `uint32` directly?

``` go
if uint64(ipArg) > 0xffffffff{
    ...
}
uIPArg := uint32(ipArg)
```